### PR TITLE
docs and opts

### PR DESCRIPTION
**Pull request by JustinSGray**
small cleanup of GA docs and fix to a non-consistent class name

### CODE REVIEW

File: openmdao/drivers/scipy_optimizer.py [hunk 1]
```diff
@@ -534,3 +534,21 @@ def _reraise(self):
         exc = self._exc_info
         self._exc_info = None
         reraise(*exc)
+
+
+class ScipyOptimizer(ScipyOptimizerDriver):
+    """
+    Deprecated.  Use PETScKrylov.
```
Comment (naylor-b) [inline openmdao/drivers/scipy_optimizer.py]: bad comment

File: openmdao/drivers/scipy_optimizer.py [hunk 2]
```diff
@@ -534,3 +534,21 @@ def _reraise(self):
...
+    def __init__(self, **kwargs):
+        """
+        Initialize attributes.
+
+        Parameters
+        ----------
+        kwargs : dict
+            Named args.
+        """
+        super(PetscKSP, self).__init__(**kwargs)
```
Comment (naylor-b) [inline openmdao/drivers/scipy_optimizer.py]: wrong class for super

File: openmdao/drivers/scipy_optimizer.py [hunk 3]
```diff
@@ -534,3 +534,21 @@ def _reraise(self):
...
+        warn_deprecation('ScipyOptimizer is deprecated.  Use ScipyOptimizerDriver instead.')
```
Comment (naylor-b) [inline openmdao/drivers/scipy_optimizer.py]: no deprecation test

File: openmdao/drivers/scipy_optimizer.py (hunk 2 above)
Comment (Kenneth-T-Moore) [reply openmdao/drivers/scipy_optimizer.py]: Also need to add something like:
        warn_deprecation("'MultiFiMetaModel' component has been deprecated. Use"
                         "'MultiFiMetaModelUnStructured' instead.")
to the init.

File: openmdao/api.py [hunk 4]
```diff
@@ -66,7 +66,7 @@
     from openmdao.drivers.pyoptsparse_driver import pyOptSparseDriver
 except ImportError:
     pass
-from openmdao.drivers.scipy_optimizer import ScipyOptimizer
+from openmdao.drivers.scipy_optimizer import ScipyOptimizer, ScipyOptimizerDriver
```
Comment (swryan) [inline openmdao/api.py]: a small difference maybe, but the suggestion was `ScipyOptimizeDriver`, as in a driver based on `scipy.optimize`  (nor `r` on Optimize)  Hey it saves you a keystroke! ;)

File: openmdao/docs/feature_reference/building_blocks/drivers/genetic_algorithm.rst [hunk 5]
```diff
@@ -3,13 +3,13 @@
 Simple GA Driver
 ****************
 
-The SimpleGADriver is a driver based on a simple genetic algorithm implementation sourced from the lecture
-notes for the class 2009 AAE550 taught by Prof. William A. Crossley. The implementation is pure Python.
+.. note:: 
+    The SimpleGA is a driver based on a simple genetic algorithm implementation sourced from the lecture
```
Comment (swryan) [inline openmdao/docs/feature_reference/building_blocks/drivers/genetic_algorithm.rst]: did you really mean to remove `Driver` from the name?  `SimpleGADriver` is a thing, `SimpleGA` is not?

File: openmdao/drivers/scipy_optimizer.py [hunk 6]
```diff
@@ -534,3 +536,22 @@ def _reraise(self):
...
+        """
+        Initialize attributes.
+
+        Parameters
+        ----------
+        kwargs : dict
+            Named args.
+        """
+        super(ScipyOptimizeDriver, self).__init__(**kwargs)
+        warn_deprecation("'ScipyOptimizer' property provides backwards compatibility "
```
Comment (swryan) [inline openmdao/drivers/scipy_optimizer.py]: delete the word `property`?

File: openmdao/drivers/tests/test_scipy_optimizer.py [hunk 7]
```diff
@@ -16,7 +17,20 @@
 from openmdao.test_suite.groups.sin_fitter import SineFitter
 
 
-class TestScipyOptimizer(unittest.TestCase):
+class TestScipyOptimizeDriver(unittest.TestCase):
+
+    def test_scipyoptimizer_deprecation(self): 
+
+        msg = "'ScipyOptimizer' property provides backwards compatibility " \
```
Comment (swryan) [inline openmdao/drivers/tests/test_scipy_optimizer.py]: delete the word `property`?

**Pull request merged**